### PR TITLE
[Cognitive Services Language] TA April preview SpeedUnit spelling fix

### PR DESCRIPTION
**Pull request by quentinRobinson**
# Data Plane API - Pull Request
SpeedUnit type renaming to be plural.

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*API Spec engagement record*](https://aka.ms/azsdk/onboarding/restapischedule).

* Link to API Spec engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [x] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous API Spec document (if applicable), and the root paths that have been updated. </sup>
* Design Document:
* Previous API Spec Doc:
* Updated paths:

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azsdk/onboarding/restapischedule) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azsdk/onboarding/restapischedule).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

**Review by heaths — CHANGES_REQUESTED** [review]
You need to change the snapshot under the specification folder too.

**Comment by heaths** [comment]
The breaking change is only within the same version, so not actually breaking.

/cc @weshaggard @maririos for another example

File: dev/cognitiveservices/data-plane/Language/analyzetext.json [hunk 1]
```diff
@@ -2549,19 +2549,19 @@
...
-        "KilometerPerSecond",
-        "MilePerHour",
+        "MetersPerSecond",
+        "KilometersPerHour",
+        "KilometersPerMinute",
+        "KilometersPerSecond",
+        "MilesPerHour",
         "Knot",
         "FootPerSecond",
         "FootPerMinute",
```
Comment (joseharriaga) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:2559]: Should these be plural too? "Knots", "FeetPerSecond", and "FeetPerMinute"?

Comment (quentinRobinson) [reply dev/cognitiveservices/data-plane/Language/analyzetext.json]: Updated!

File: dev/cognitiveservices/data-plane/Language/analyzetext.json [hunk 2]
```diff
@@ -2549,19 +2549,19 @@
...
+        "KilometersPerSecond",
+        "MilesPerHour",
+        "Knots",
+        "FeetPerSecond",
+        "FeetPerMinute",
+        "YardsPerMinute",
+        "YardsPerSecond",
+        "MetersPerMillisecond",
+        "CentimetersPerMillisecond",
+        "KilometersPerMillisecond"
```
Comment (kristapratico) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:2564]: Just want to make sure that we're okay with diverging from what's defined in CLU. We won't be able to share these definitions between TA/CLU.

https://github.com/Azure/azure-rest-api-specs/blob/756495dd7e0e2f5181039def47a8c85ff0787b66/specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzeconversations.json#L980-L998

Comment (quentinRobinson) [reply dev/cognitiveservices/data-plane/Language/analyzetext.json:2564]: Yes, we are aware we are diverging from the CLU definition.

**Comment by weshaggard** [comment]
> The breaking change is only within the same version, so not actually breaking.
> 
> /cc @weshaggard @maririos for another example

cc @konrad-jamrozik

**Review by heaths — APPROVED** [review]

**Comment by konrad-jamrozik** [comment]
> The breaking change is only within the same version, so not actually breaking.
> 
> /cc @weshaggard Wes Haggard FTE @maririos Mariana Rios Flores FTE for another example

@heaths can you let me know which of the reported breaking change errors is the false positive? Or all of them? There are many.

**Comment by heaths** [comment]
All of them. This version hasn't released yet, so how can it have breaking changes against itself? This has long been a problem and, as you said, there are many so it takes me a lot of time throughout a week to thoroughly and satisfactorily investigate breaking changes to know if I can override.

**Comment by konrad-jamrozik** [comment]
Thank you @heaths. I am aggregating all such similar issues in comments of this issue:

- https://github.com/Azure/azure-sdk-tools/issues/5943

**Pull request merged**